app/src/main/java/com/arctouch/codechallenge/view/splash/SplashViewModel.kt

```diff
@@ -0,0 +1,6 @@
+package com.arctouch.codechallenge.view.splash
+
+import com.arctouch.codechallenge.view.util.BaseViewModel
+
+class SplashViewModel() : BaseViewModel() {
```

Comment:
Are you using this somehow?

Reply:
At first moment isn't being used but for my idea is use this to update information about genres and retrieve and update movies saved at Room